app/posts/manage-teacher-training-applications/2020-12-09-full-time-part-time.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Filter applications by full time or part time
+description: Provide options to view the application list by full time or part time courses
+date: 2020-12-09
+---
+
+This post documents the introduction of the full time or part time filter alongside existing filters on the applications list.
```

Comment:
I think for this post, I think we don't need a separate intro and how it works sections.

Points are:

1. We added a filter
2. Labelled it full or part time to avoid jargon
3. added the study mode to the application card